mycroft/util/format.py

```diff
@@ -454,83 +468,256 @@ def nice_year(dt, lang=None, bc=False):
     return date_time_format.year_format(dt, full_code, bc)
 
 
-def nice_duration(duration, lang=None, speech=True):
+def duration_handler(time1, lang=None, speech=True, *, time2=None,
```

Comment:
Let's mark this as an internal function by naming it `_duration_handler` then we can change the parameters and thing as we like without worrying about keeping the api stable.

Reply:
Pushed.